src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import { Link, Route, Routes } from "react-router-dom";
+
+import Home from "./pages/Home";
+import About from "./pages/About.";
+
+function App() {
+  return (
+    <>
+      <nav>
```

Comment:
@Angemichel12 Instead of rendering nav like this, you can try using layout with outlet from react router dom